src/migrations.zig

```diff
@@ -8,24 +8,41 @@ var arenaAllocator = main.utils.NeverFailingArenaAllocator.init(main.globalAlloc
 const migrationAllocator = arenaAllocator.allocator();
 
 var blockMigrations: std.StringHashMap([]const u8) = .init(migrationAllocator.allocator);
-
-pub fn registerBlockMigrations(migrations: *std.StringHashMap(ZonElement)) void {
-	std.log.info("Registering {} block migrations", .{migrations.count()});
-
+var biomeMigrations: std.StringHashMap([]const u8) = .init(migrationAllocator.allocator);
+
+const MigrationType = enum {
+	block,
+	biome,
+};
+
+pub fn registerAll(comptime typ: MigrationType, migrations: *std.StringHashMap(ZonElement)) void {
+	std.log.info("Registering {} biome migrations", .{migrations.count()});
+	const collection = switch(typ) {
+		.block => &blockMigrations,
+		.biome => &biomeMigrations,
+	};
 	var migrationIterator = migrations.iterator();
 	while(migrationIterator.next()) |migration| {
-		register(&blockMigrations, "block", migration.key_ptr.*, migration.value_ptr.*);
+		register(typ, collection, migration.key_ptr.*, migration.value_ptr.*);
 	}
 }
 
 fn register(
+	comptime typ: MigrationType,
 	collection: *std.StringHashMap([]const u8),
-	assetType: []const u8,
 	addonName: []const u8,
 	migrationZon: ZonElement,
 ) void {
-	if((migrationZon.toSlice().len == 0)) {
-		std.log.err("Skipping incorrect {s} migration data structure from addon {s}", .{assetType, addonName});
+	if(migrationZon != .array) {
+		if(migrationZon == .object and migrationZon.object.count() == 0) {
+			std.log.info("Skipping empty {s} migration data structure from addon {s}", .{@tagName(typ), addonName});
```

Comment:
As requested in my last review, this should be a warning.

Reply:
```suggestion
			std.log.warn("Skipping empty {s} migration data structure from addon {s}", .{@tagName(typ), addonName});
```